docs/source/receptorctl/receptorctl_status.rst

```diff
@@ -0,0 +1,171 @@
+--------------------------
+Receptorctl status command
+--------------------------
```

Comment:
I'd probably change this to just `status` or maybe `status` command.

With this "Receptorctl x command" format it will get difficult to parse through the doc at a high level because all the sections will look the same.

```
Receptorctl status command
Receptorctl foo command
Receptorctl bar command
```

The preceding format results in the command getting lost in the surrounding text, which means the reader needs to work a bit harder. From the title, we've got the context that we're talking about Receptorctl client commands. So a simpler title for each section would make more sense to me.

```
status
foo
bar
```